www/components/pages/Home/Inbox/Inbox.js

```diff
@@ -23,17 +50,75 @@ export default function Inbox({ meetings, refresh }) {
     );
   });
 
-  if (lineItems.length) {
-    return <div className={styles.table}>{lineItems}</div>;
+  const handleNameChange = (event) => {
+    setFilters((prevState) => ({ ...prevState, name: event.target.value }));
+    setFetchingMeetings(true);
+  };
+
+  const handleOwnersChange = (event, value) => {
+    setFilters((prevState) => ({ ...prevState, owners: value.owner_id }));
+  };
+
+  const users = [
+    ...new Set(
+      meetings.map((item) => ({
+        name: item.owner.email,
+        owner_id: item.owner._id,
+      }))
+    ),
```

Comment:
Sets unfortunately don't work for objects, so this will allow duplicates. You could do something like this though.

```js
const users = Object.values(
  meetings.reduce((acc, meeting) => {
    acc[meeting.name] = meeting;

    return acc;
  }, {})
);
```